src/Bootstrap/CDN.elm

```diff
@@ -36,7 +36,7 @@ stylesheet =
         []
 
 
-{-| For your convenience you can also include font-awesome as an inline node. Worth knowing that this `Bootstrap for Elm` has no dependencies on font awesome
+{-| Font Awesome can also be conveniently included as an inline node. Font Awesome does not depend on `Bootstrap for Elm`.
```

Comment:
Oops, you're right. What do you think of styling `Bootstrap for Elm` as `elm-bootstrap` instead?